<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>940</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1320</width>
    <height>940</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(247, 249, 252);</string>
  </property>
  <widget class="QWidget" name="LeftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>255</width>
     <height>940</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 243, 246);</string>
   </property>
   <widget class="QPushButton" name="recommend">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>112</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/recommend.png);
image-position: left;
border: none;
</string>
    </property>
    <property name="text">
     <string>推荐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selected">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>160</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/selected.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>精选</string>
    </property>
   </widget>
   <widget class="QPushButton" name="podcast">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>208</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/podcast.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>播客</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roam">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>256</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/roam.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>漫游</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dynamics">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>304</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/dynamics.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>动态</string>
    </property>
   </widget>
   <widget class="QWidget" name="Theme" native="true">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>20</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/new/prefix1/log.png)</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>374</y>
      <width>173</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:  none;
background-color: rgba(184, 189, 198, 0.5);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>409</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(160, 166, 177);</string>
    </property>
    <property name="text">
     <string>我的</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownloadManagement">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>587</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/DownloadManagement.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>下载管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LocalMusic">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>635</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/LocalMusic.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>本地音乐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="collect">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>539</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/collect.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>我的收藏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecentlyPlayed">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>491</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/RecentlyPlayed.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>最近播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myLike">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>443</y>
      <width>196</width>
      <height>45</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/myLike.png);
image-position: left;
border: none;</string>
    </property>
    <property name="text">
     <string>我喜欢的音乐</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="retreat">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>20</y>
     <width>37</width>
     <height>47</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/return.png);
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="DownWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>840</y>
     <width>1320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top: 1px solid rgba(184, 189, 198, 0.5);</string>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>13</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 12px solid rgb(4, 4, 4);
border-radius: 37px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SongInformation">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>23</y>
      <width>226</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;</string>
    </property>
    <property name="text">
     <string>如果爱忘了（live）-  汪苏泷/单依纯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddToPlaylist">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>50</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/AddToPlaylist.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewComments">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>50</y>
      <width>36</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/ViewComments.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="share">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/share.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/download.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="CentreDownWidget" native="true">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>0</y>
      <width>474</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>15</y>
       <width>54</width>
       <height>54</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/pause.png);
border: none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="like">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>25</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/like.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PreviousSong">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>25</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/PreviousSong.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="NextSong">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>25</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/NextSong.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ListLoop">
     <property name="geometry">
      <rect>
       <x>339</x>
       <y>25</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/ListLoop.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="progressSlider">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>70</y>
       <width>390</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {
    border: none;
    margin-left: 8px;
    margin-right: 8px;
}

/* 默认水平滑块的样式 */
QSlider::handle:horizontal {
    background: transparent;
    width: 16px;
    height: 16px;
    margin: -6px 0;
    border-radius: 8px;
    transition: all 0.3s ease; /* 添加过渡效果 */
}

/* 悬停时水平滑块的样式 */
QSlider::handle:horizontal:hover {
    background: rgb(0, 0, 0); /* 悬停时滑块颜色变为黑色 */
    width: 16px;
    height: 16px;
    margin: -6px 0;
    border-radius: 8px;
    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.3); /* 添加阴影效果 */
}

/* 滑轨的样式 */
QSlider::groove:horizontal {
    background: rgb(242, 107, 130);
    height: 5px;
    border-radius: 2px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RightDownWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>0</y>
      <width>320</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QPushButton" name="DesktopLyrics">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/DesktopLyrics.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="MemberSoundEffects">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/MemberSoundEffects.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="LetIsStartListeningTogether">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/LetIsStartListeningTogether.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="VolumeControl">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/VolumeControl.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="PlayList">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/PlayList.png);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="RightTopWidget" native="true">
   <property name="geometry">
    <rect>
     <x>738</x>
     <y>2</y>
     <width>581</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <widget class="QPushButton" name="Settings">
    <property name="geometry">
     <rect>
      <x>333</x>
      <y>20</y>
      <width>37</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/Settings.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>20</y>
      <width>37</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/close.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="skinPeeler">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>37</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/skin-peeler.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mail">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>20</y>
      <width>37</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/mail.png);
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="maximize">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>37</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/maximize.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>20</y>
      <width>37</width>
      <height>47</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/minimize.png);
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>414</x>
      <y>32</y>
      <width>1</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:  none;
background-color: rgba(184, 189, 198, 0.5);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>24</y>
      <width>39</width>
      <height>39</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(226, 229, 233);
border-radius: 19px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="username">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>34</y>
      <width>62</width>
      <height>19</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>璐璐家投了</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vip">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>33</y>
      <width>38</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/vip.png);
border-radius: 10px;
border-top-right-radius: 0;
border-bottom-right-radius: 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="vipText">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>33</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 42, 42);
border-radius: 10px;
border-top-left-radius: none;
color: rgb(236, 212, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Personal_Information">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>35</y>
      <width>19</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/Personal_Information.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>22</y>
     <width>58</width>
     <height>44</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(226, 229, 233);
border-radius: 10px;
border-right: none;
border-top-right-radius: none
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>683</x>
     <y>22</y>
     <width>44</width>
     <height>44</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(226, 229, 233);
border-radius: 10px;
image: url(:/new/prefix1/search_for.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>22</y>
     <width>264</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(226, 229, 233);
border-radius: 10px;
border-left: none;
border-top-left-radius: none</string>
   </property>
  </widget>
  <widget class="QWidget" name="MainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>89</y>
     <width>1065</width>
     <height>751</height>
    </rect>
   </property>
  </widget>
  <zorder>search</zorder>
  <zorder>retreat</zorder>
  <zorder>LeftWidget</zorder>
  <zorder>DownWidget</zorder>
  <zorder>RightTopWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>MainWidget</zorder>
 </widget>
 <tabstops>
  <tabstop>retreat</tabstop>
  <tabstop>recommend</tabstop>
  <tabstop>selected</tabstop>
  <tabstop>podcast</tabstop>
  <tabstop>dynamics</tabstop>
  <tabstop>roam</tabstop>
  <tabstop>DownloadManagement</tabstop>
  <tabstop>LocalMusic</tabstop>
  <tabstop>collect</tabstop>
  <tabstop>RecentlyPlayed</tabstop>
  <tabstop>myLike</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
